bundles/GoogleMarketingBundle/public/js/startup.js

```diff
@@ -0,0 +1,35 @@
+pimcore.registerNS('pimcore.bundle.googlemarketing.startup');
+
+pimcore.bundle.googlemarketing.startup = Class.create({
+    initialize: function () {
+        document.addEventListener(pimcore.events.preMenuBuild, this.preMenuBuild.bind(this));
+    },
+
+    preMenuBuild: function (event) {
+        const menu = event.detail.menu;
+        const user = pimcore.globalmanager.get('user');
+        const perspectiveCfg = pimcore.globalmanager.get("perspective");
+
+        if (menu.marketing && perspectiveCfg.inToolbar("settings.marketingReports")
+            && user.isAllowed("reports") && user.isAllowed("system_settings")) {
```

Comment:
```suggestion
            && user.isAllowed("google_marketing") && user.isAllowed("system_settings")) {
```
`reports` permission is used for custom reports

Reply:
We do need a migration then right, if a user has reports and system_settings he will get google_marketing permissions too?